src/main/java/com/board/config/boardconfiggui/controllers/SlaveWidgetController.java

```diff
@@ -0,0 +1,101 @@
+package com.board.config.boardconfiggui.controllers;
+
+import com.board.config.boardconfiggui.common.Utils;
+import com.board.config.boardconfiggui.ui.models.SlaveDeviceConfigModel;
+import javafx.collections.FXCollections;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.fxml.Initializable;
+
+import java.net.URL;
+import java.util.Arrays;
+import java.util.ResourceBundle;
+
+public class SlaveWidgetController implements Initializable {
+    @FXML
+    private LabelTextFieldWidgetController i2cFmPlusSpeedController;
+
+    @FXML
+    private LabelTextFieldWidgetController addressController;
+
+    @FXML
+    private LabelTextFieldWidgetController hdrCapableController;
+    @FXML
+    private LabelTextFieldWidgetController legacyI2CDevController;
+    @FXML
+    private LabelTextFieldWidgetController hasStaticAddressController;
+    @FXML
+    private LabelTextFieldWidgetController staticAddressController;
+    @FXML
+    private LabelTextFieldWidgetController dynamicAddressController;
+    @FXML
+    private LabelTextFieldWidgetController isIbiDeviceController;
+    @FXML
+    private LabelTextFieldWidgetController ibiPayloadSizeController;
+    @FXML
+    private LabelTextFieldWidgetController ibiPayloadSpeedLimitController;
+
+    @FXML
+    private LabelComboBoxWidgetController devRoleController;
+
+    private SlaveDeviceConfigModel slaveDeviceConfigModel;
+
+    public void setSlaveWidgetObject() {
+        //TODO: need to get the object from maim class and update the UI
+    }
+
+    @Override
+    public void initialize(URL url, ResourceBundle resourceBundle) {
+        slaveDeviceConfigModel = new SlaveDeviceConfigModel("Device0");
```

Comment:
The name of the slave should be collected from the user, it shouldn't be the hardcoded value "Device0 "